web/public/data/kgrams.csv

```diff
@@ -1,57 +1,86 @@
 id,hash,data,files
-8,284608,,"[4,6,7]"
-9,3206193,,"[6,4,7]"
-10,1296870,,"[6,4,7]"
-11,155427,,"[6,4,7]"
-12,376343,,"[6,4,7]"
-13,1581299,,"[6,5,7]"
-14,756037,,"[6,5,7]"
-15,2178801,,"[6,5,7]"
-16,932740,,"[6,5,7]"
-17,1225437,,"[6,5,7]"
-18,222159,,"[6,5,7]"
-19,3899861,,"[6,7]"
-20,774902,,"[6,7]"
-21,1238435,,"[6,4,7]"
-22,2597709,,"[6,7]"
-23,460538,,"[6,7]"
-24,987802,,"[6,4,7]"
-25,2104191,,"[6,4,7]"
-26,2573676,,"[6,4,7]"
-27,13644099,,"[7,6]"
-28,13157048,,"[7,6]"
-29,10058977,,"[7,6]"
-30,2364626,,"[7,6]"
-31,575218,,"[7,6]"
-32,519007,,"[7,6]"
-33,3038801,,"[7,6]"
-34,2279188,,"[7,6]"
-35,980786,,"[7,6]"
-36,2349113,,"[7,6]"
-37,3032394,,"[7,6]"
-38,4461448,,"[7,6]"
-39,2555132,,"[7,6]"
-40,807031,,"[7,6]"
-41,1792882,,"[7,6]"
-42,3017714,,"[7,6]"
-43,1180646,,"[7,6]"
-44,72169,,"[7,6]"
-45,879920,,"[7,6]"
-46,669391,,"[7,6]"
-47,2420877,,"[7,6]"
-48,704024,,"[7,6]"
-49,563797,,"[7,6]"
-50,6648720,,"[7,6]"
-51,4629770,,"[7,6]"
-52,637251,,"[7,6]"
-53,3037355,,"[7,6]"
-54,1819788,,"[7,6]"
-55,651621,,"[7,6]"
-56,411467,,"[7,6]"
-57,4788680,,"[7,6]"
-58,3995583,,"[7,6]"
-59,2683139,,"[7,6]"
-60,581367,,"[7,6]"
-61,1077799,,"[7,6]"
-62,3366137,,"[7,6]"
-63,165354,,"[7,6]"
+8,1083181,,"[6,4,7]"
+9,2676335,,"[6,4,7]"
+10,6644131,,"[6,4,7]"
+11,604601,,"[6,4,7]"
```

Comment:
Do we want to change the public/data files?

Reply:
These changes are needed for dynamic fragment loading. All changes have been discussed an approved by @rien